LSHandler.js

```diff
@@ -0,0 +1,30 @@
+class LSHandler {
+    constructor() {
+        if (localStorage.userId == undefined)
+            localStorage.userId = 1
+    }
+
+    ifUserIsInLocal = (localArray, currentData) => localArray.every((data) => data.user.user.results[0].name.first !== currentData.user.results[0].name.first && data.user.user.results[0].name.last !== currentData.user.results[0].name.last && data.user.pokemon.id !== currentData.pokemon.id)
+
+    getData = () => JSON.parse(localStorage.data || "{}")
+
+    ifObjectIsEmpty = (obj) => (Object.keys(obj).length === 0 && obj.constructor === Object)
```

Comment:
if there is only one perameter you do not need to use `()`
`ifObjectIsEmpty = obj => Object.keys...`